locale/bg.po

```diff
@@ -7,15 +7,15 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2020-12-17 20:40+0200\n"
+"POT-Creation-Date: 2020-12-18 00:41-0800\n"
 "PO-Revision-Date: 2020-12-16 02:36+0000\n"
 "Last-Translator: Georgi Georgiev (Жоро) <g.georgiev.shumen@gmail.com>\n"
 "Language-Team: Bulgarian <https://translate.revolutionarygamesstudio.com/"
 "projects/thrive/thrive-game/bg/>\n"
-"Language: bg\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Language: bg\n"
```

Comment:
This change doesn't seem to go away. This is not something that happens on my computer. (at least when trying to master)
If  you can't make the change go away, you'll need to manually revert this, because babel on my computer wants the Language key to be after Language-Team key, and won't let this change be.